Dockerfile

```diff
@@ -1,6 +1,6 @@
 # quay.io/vouch/vouch-proxy
 # https://github.com/vouch/vouch-proxy
-FROM golang:1.18 AS builder
+FROM golang:1.20 AS builder
```

Comment:
I would directly use `golang:1.21`

Reply:
Sure. I wasn't sure how close to the bleeding edge to go :) since 1.21 was only just released. But, I'm  happy to bump this to 1.21. I will push the updates in a few minutes.